### DISCUSSION TITLE
Stream Start Failed with IP camera RTSP link.

### DISCUSSION THREAD

**Discussion by jaisaravanan-dev** [ant-media/Ant-Media-Server#6027] [Q&A]
rtsp://192.168.144.25:8554/main.264

I have mentioned the RTSP link above, but I can't broadcast my IP camera RTSP link to the ant media server and I am getting the below-mentioned error. When I have tried with VLC the above link is working fine.

> Stream Start Failed.
> Error: Stream is already active. It's already streaming or trying to connect

I have checked with the test RTSP link from https://rtsp.stream/ it working fine.

Could any one help in this issue

**Comment by USAMAWIZARD** [comment]
Hi @JaiSaro ,
can you please provide the output for this command
ffprobe RTSP_URL

please run the below commands and let me know if both of them works or any one of them fails
ffplay -rtsp_transport tcp -i RTSP_URL
ffplay -rtsp_transport udp -i RTSP_URL

Which version of Ant Media are you using right now.
